scss/_variables.scss

```diff
@@ -583,7 +583,7 @@ $border-radius-pill:          50rem !default;
 // scss-docs-end border-radius-variables
```

Comment:
Any change needed from line 565 to 568?

Reply:
Hmm good question 🤔 I'll try if it works. Good catch! I thought it was done like this in Bootstrap but in fact they're using fixed values and not vars.

Reply:
Can't work because `.border-2` would be defined like this: `--bs-border-width: var(--bs-border-width);`